src/logion/services/locauthorization.service.ts

```diff
@@ -14,9 +14,9 @@ export class LocAuthorizationService {
         private polkadotService: PolkadotService,
     ) {}
 
-    async ensureContributor(httpRequest: Request, request: LocRequestAggregateRoot, allowInvitedContributor = false): Promise<SupportedAccountId> {
+    async ensureContributor(httpRequest: Request, request: LocRequestAggregateRoot, allowInvitedContributor = false, allowOwner = true): Promise<SupportedAccountId> {
```

Comment:
Would be nice to encapsulate those flags in a VO with some explicit static factory methods (e.g. `locContributor`, `itemContributor`, ...).

Reply:
Good suggestion, thanks.
Done in [#749db6f](https://github.com/logion-network/logion-backend-ts/pull/295/commits/749db6f421d6f8da03b6a53513fc6146ec43ef59)